src/main/java/net/sf/jabref/gui/externalfiles/FindFullTextAction.java

```diff
@@ -91,10 +91,11 @@ public void update() {
                         basePanel.getBibDatabaseContext(), entry);
                 try {
                     def.download(result.get(), file -> {
-                        FileListTableModel tm = new FileListTableModel();
-                        entry.getField(FieldName.FILE).ifPresent(tm::setContent);
-                        tm.addEntry(tm.getRowCount(), file);
-                        String newValue = tm.getStringRepresentation();
+                        FileListTableModel fileLinkModel = new FileListTableModel();
+                        entry.getField(FieldName.FILE).ifPresent(fileLinkModel::setContent);
+                        // add full text file link at first position
+                        fileLinkModel.addEntry(0, file);
```

Comment:
What happens if there is already a file linked?
Will the linked file then be moved one down?